terraform/nextstrain_s3.tf

```diff
@@ -0,0 +1,18 @@
+resource "aws_s3_bucket" "nextstrain_jobs" {
+  bucket = "aspen-nextstrain-jobs"
+  acl = "private"
+
+  # this is what nextstrain docs reccomend
+  lifecycle_rule {
+    enabled = true
+
+    transition {
+      days = 30
+      storage_class = "GLACIER"
+    }
```

Comment:
i'd just punt to ONEZONE_IA.  Glacier is such a pain to retrieve anyway.

Reply:
cool i didn't know about ONEZONE_IA